### ISSUE TITLE
Please restore symlinks for base images

### ISSUE THREAD

**Issue by hwoarang** [openSUSE/docker-containers-build#29]
It seems like #28 removed the various ``openSUSE-Leap-42.3.base.$arch.tar.xz`` symlinks and this breaks some OpenStack services which were depending on them. Could you please restore them?

**Comment by hwoarang** [comment]
@davidcassany @Vogtinator

**Comment by davidcassany** [comment]
@hwoarang I am not sure I understand the issue. So the problem you have is that `openSUSE-Leap-42.3.base.$arch.tar.xz` tarballs are no longer downloadable from this repo? I am sorry if this caused you issues, I was not aware there was anyone using this repo for anything else than submitting to the official docker library. In fact, this repository was created with the only purpose of submitting images to the official library.

AFAICT this tarball naming convention was only applied to automate the submission to the official docker library within our own internal CI and workflow. Since we are not using this CI anymore we just felt free to modify tarball names, as we do not rely in any convention anymore.

Probably @fvogt can give you further advise on how this will look like in the future and establish some criteria, but I still doubt this is the best way to proceed. Couldn't you rely on what is in [docker-library/official-images/library/opensuse](https://github.com/docker-library/official-images/blob/master/library/opensuse) and extract all the information you need (like were are the tarballs and how are named) from there? After all this is the official publicly available data to reproduce what you'd get with a `docker pull opensuse:leap`.

**Comment by hwoarang** [comment]
@davidcassany yeah we were using ``openSUSE-Leap-42.3.base.$arch.tar.xz`` in OpenStack in order to get some base rootfs for LXC containers. We are using these outside of the docker context similar to

``https://github.com/CentOS/sig-cloud-instance-images/raw/CentOS-7/docker/centos-7-docker.tar.xz``
and
``http://cdimage.ubuntu.com/ubuntu-base/releases/16.04/release/ubuntu-base-16.04.4-base-x86_64.tar.gz``

so we were hoping for some static filenames for openSUSE too. If that's not possible then I can simply download and parse the tarball build number from the Dockerfile

**Comment by Vogtinator** [comment]
> so we were hoping for some static filenames for openSUSE too. If that's not possible then I can simply download and parse the tarball build number from the Dockerfile

This repository won't be maintained in the future - it's only kept until 42.3 is EOL to submit security updates.

I recommend you to build an image specifically for your purpose - you can start by branching the official containers on OBS. For static filenames, you can simply use a `kiwi_post_run` hook to rename the tarball.

**Comment by hwoarang** [comment]
I see. Sadly building such tarballs is outside the scope of our projects so we were looking to simply consume something published from distributions directly. Thank you for the reply. We will try to look for alternative rootfs tarballs for our containers.

**Comment by Vogtinator** [comment]
> Sadly building such tarballs is outside the scope of our projects so we were looking to simply consume something published from distributions directly.

Sounds like something openSUSE could provide officially - can you file a bug report with your requirements?
